acct.bionimbus.org/manifest.json

```diff
@@ -22,8 +22,6 @@
     "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2023.01",
     "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2023.01",
     "guppy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/guppy:2023.01",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2023.01",
```

Comment:
we should probably remove the `manifests/hatchery/` directory for all of there too

Reply:
Fair. ICGC & GenoMEL didn't have them, but added the other removals to this PR.